linera-core/src/updater.rs

```diff
@@ -370,6 +360,25 @@ where
         Ok(())
     }
 
+    async fn send_chain_information_to_chain_heights(
```

Comment:
@andresilva91 @afck What is this name supposed to mean?

Reply:
`send_chain_information_up_to_chain_heights`?

Reply:
Good point, it should be "up to".

And maybe just `send_chain_info_up_to_heights`? No need to repeat "chain". And we already use the abbreviation "info" in the type itself.

Reply:
Good point! I can send a PR renaming it on Monday

Reply:
https://github.com/linera-io/linera-protocol/pull/2859